tests/ui/error-codes/E0396-fixed.rs

```diff
@@ -1,9 +0,0 @@
-#![feature(const_mut_refs)]
```

Comment:
Shouldn't these be `//@ check-pass` instead of removed?

Reply:
I removed it since the error code E0396 does not exist any more, so there can't be an error code test for it that makes sense...
We have other tests already covering the exact same thing, like `tests/ui/consts/const-deref-ptr`.